test/parallel/test-console-sync-write-error.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
I’ve updated this with tests for something that errors asynchronously… the inner workings of console aren’t really tied to I/O at all, so I’m not sure it makes sense to add explicit tests for these